src/unframed/d2lfetch-auth.js

```diff
@@ -2,6 +2,8 @@ const D2L_FETCH_CACHED_TOKENS = {},
 	D2L_FETCH_IN_FLIGHT_REQUESTS = {},
 	STORAGE_NAME = 'D2L.Fetch.Tokens';
 
+let D2L_FETCH_CLOCK_SKEW = 0;
```

Comment:
Theoretically might want to localstorage this. But I'll let @dlockhart do that if he wants, since he likes localStorage so much.

Reply:
Also seems reasonable to re-calculate it in case it fixes itself somehow? I dunno.

Reply:
Recalculated every time we fetch a token